pkg/broker/broker.go

```diff
@@ -76,23 +82,42 @@ func (br *Broker) ListenForClients() {
 		case sb := <-br.NewClients:
 			br.clients[sb] = true
 			cc, ok := br.clientsCount.Load(sb.ID)
-			if ok {
+			if ok && cc.(int) > 0 {
```

Comment:
We should check here that the conversion works successfully. If we don't do so, this may panic, which is definitely unwanted.

Reply:
Thanks for clarifying, fixed.